voxblox_ros/include/voxblox_ros/tsdf_server.h

```diff
@@ -127,8 +127,12 @@ class TsdfServer {
   // What output information to publish
   bool publish_tsdf_info_;
   bool publish_slices_;
+  bool publish_pointclouds_;
   bool publish_tsdf_map_;
 
+  // Whether to save the latest mesh message sent (for inheriting classes).
+  bool cache_mesh_;
```

Comment:
nit: `cached_mesh_`?

Reply:
Nope, it's a present tense verb as it's a setting: whether to *cache* the mesh, not whether it's already been cached.

Reply:
you are right of course, must have mixed it up with the other member that actually stores the mesh